map/framework.cpp

```diff
@@ -2178,6 +2178,14 @@ place_page::Info Framework::BuildPlacePageInfo(place_page::BuildInfo const & bui
     // Selection circle should match feature
     FillFeatureInfo(selectedFeature, outInfo);
 
+    if (!outInfo.IsBookmark() && !isBuildingSelected)
```

Comment:
Does a bookmark on a building work with this check?

Reply:
Building centroid works as POI. If you tap on centroid or on a building label then `isBuildingSelected==false` and bookmark is found correctly.

If you tap inside a building then PlacePage shows building properties but with no bookmark.

You can install and try 😉

Reply:
Thanks for the details.

> You can install and try 😉

Building, installing, and trying every PR takes a lot of time, and is not possible from iPhone. I'm doing it, of course, when there's some time.